backend/dataall/base/aws/parameter_store.py

```diff
@@ -37,6 +37,18 @@ def get_parameter_value(AwsAccountId=None, region=None, parameter_path=None):
             raise Exception(e)
         return parameter_value
 
+    @staticmethod
+    def get_parameters_by_path(AwsAccountId=None, region=None, parameter_path=None):
```

Comment:
What does this function do?

Reply:
This function is used to fetch all the parameters belonging to a hierachy of path name. For e.g. /path1/param1 and /path1/param2, this will fetch all the path under the /path1 

This is used to get the names of all the scheduled ECS tasks created in data.all